### DISCUSSION TITLE
Automatic1111 webui

### DISCUSSION THREAD

**Discussion by smokeyhallow** [hpcaitech/ColossalAI#1934] [Community | General]
Starting a discussion on how to get this working with the webui from Automatic1111 considering its the most popular SD project.

**Comment by Miraculix200** [comment]
Someone would need to code an extension for it

**Comment by DonStroganotti** [comment]
I'd like to know if anyone has even got it to work as is with a 8GB GPUs as they claim, with any sort of performance. 

If that's possible it makes sense to focus development efforts on implementing this alongside or instead of the dreambooth extension.

**Comment by binmakeswell** [comment]
Hi @smokeyhallow @Miraculix200 Thank you for the feedback.
https://github.com/AUTOMATIC1111/stable-diffusion-webui
In the first stage, we are mainly optimizing training performance.
We will increase the priority of application.
